server/routes/dataReport.ts

```diff
@@ -0,0 +1,353 @@
+/*
```

Comment:
Same as above- 2 space indentation

Reply:
The two APIs calls are not only for on-demand reports but also for the scheduled reports. Once the user decides to schedule a report, only the metadata will be created. Then when he needs to download, the CSV will be generated from the metadata then return it to the client without saving it.